CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Release 0.25.6
```

Comment:
0.25.5 isn't out yet, so this can go below in the 0.25.5 section

Reply:
Must have selected the wrong source branch when checking the changelog, moved to 0.25.5